style.css

```diff
@@ -0,0 +1,124 @@
+* {
+    margin: 0; 
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 10px;
+    background-color: rgb(221, 255, 244);
+}
+
+.virtualDisplay {
+    top: 20px;
+    display: flex;
+    margin: 0 auto;
+    font-family: "Roboto";
+    width: 924px;
+    position: relative;
+    padding-left: 48px;
+}
+
+.printedText {
+    width: 864px;
+    height: 200px;
+    border-radius: 5px;
+    font-size: 2rem;
+    padding: 10px 20px;
+    color: rgb(110, 48, 134);
+    outline: none;
+}
+
+.currentLanguage {
+    color: rgb(115, 114, 116);
+    font-size: 3.5rem;
+    font-weight: 900;
+    float: right;
+    padding: 2px;
+    position: relative;
+    right: 0;
+    user-select: none;
+}
+
+.keyboard {
+    background-color: rgb(43, 179, 133);
+    display: flex;
+    flex-wrap: wrap;
+    padding: 40px 30px 15px 30px;
+    width: 884px;
+    margin: auto;
+    border-radius: 5px;
+}
+
+.info {
+    display: block;
+    margin: 10px 0 0 calc(50vw - 450px);
+    border-radius: 10px;
+    color: rgb(115, 114, 116);
+    font-size: 1.6rem;
+    padding: 5px 20px;
+    text-align: left;
+    font-family: "Roboto";
```

Comment:
font-family should be defined once at the top (in html for example)

Reply:
fixed